src/Components/NeedMilk/NeedMilk.js

```diff
@@ -1,10 +1,10 @@
 import DonorCard from '../DonorCard/DonorCard';
-import {useEffect, useState} from 'react';
-import {useQuery, gql} from '@apollo/client';
+import { useEffect, useState } from 'react';
+import { useQuery } from '@apollo/client';
 import { GET_ALL_DONORS } from '../../Graphql/Queries';
 
 const NeedMilk = () => {
-  const {error, loading, data} = useQuery(GET_ALL_DONORS)
```

Comment:
I took these out to clean up files, but do we need them for error handling components?